ansible/base.yaml

```diff
@@ -1,105 +0,0 @@
-- name: base
-  hosts: validators,prometheus,loadrunners,ephemeral
-  gather_facts: yes
-  become_method: sudo
-  vars:
-    ansible_host_key_checking: false
-
-  tasks:
-    - name: Update apt cache
-      ansible.builtin.apt:
-        update_cache: yes
-        cache_valid_time: 60
-      become: yes
-    - name: restart systemd journal
```

Comment:
Is this present in the new config ? I was not able to get logs from the comet node. If they are not retrievable via journalct anymore, how can we access them?

Reply:
Retrieving logs via journalctl was in in #54